geomesa-lambda/geomesa-lambda-datastore/src/main/scala/org/locationtech/geomesa/lambda/data/LambdaQueryRunner.scala

```diff
@@ -70,72 +58,25 @@ class LambdaQueryRunner(persistence: DataStore, transients: LoadingCache[String,
           .read(Option(query.getFilter), Option(query.getPropertyNames), Option(query.getHints), explain)
     }
   }
+}
 
-  private def runMergedQuery(sft: SimpleFeatureType, query: Query, explain: Explainer): CloseableIterator[SimpleFeature] = {
```

Comment:
Oh!  Did this get factored out between this and the MergedView DS?!  If so, wow!

Reply:
yep!